package.json

```diff
@@ -20,10 +20,11 @@
     "build": "npm run build:dev",
     "build:prod": "cross-env NODE_ENV=production rollup -c",
     "build:dev": "rollup -c",
+    "build:types": "rimraf out; tsc; api-extractor-lerna-monorepo; rimraf out",
```

Comment:
Just did a fresh install of the repo and this still doesn't work on windows. I don't think `;` separated commands work. They need to be `&&`

Reply:
Done!